Testscripts/Linux/docker_compose_wordpress_mysql_app.sh

```diff
@@ -84,6 +84,10 @@ function EvaluateTestResult() {
 }
 UtilsInit
 
+if [[ $(uname -m) != "x86_64" ]];then
+    HandleSkip "The Docker Compose doesn't support other architecture except x86_64" 0
```

Comment:
do you have a link to say this?

Reply:
Have modified. Added some comments above the code line.